FilterQL.md

```diff
@@ -23,6 +24,7 @@ Comparison    = Identifier ComparisonOp Literal
 ComparisonOp  = ">" | ">=" | "<" | "<=" | "=="
 EXISTS        = "EXISTS" Identifier
 IN            = Identifier "IN" (Literal1, Literal2, ...)
+INTERSECTS    = Identifier "INTERSECTS" (Literal1, Literal2, ...)
```

Comment:
I'm not sure, but literals are all we require for now. We can always expand it later.